applications/launchpad/backend/src/main.rs

```diff
@@ -68,6 +69,7 @@ use crate::{
     grpc::WalletTransaction,
 };
 
+#[warn(clippy::too_many_lines)]
```

Comment:
It's better to break main() up into more readable helper functions. Please create an issue for this.

Reply:
Added #422